operator/Makefile

```diff
@@ -0,0 +1,401 @@
+	@echo ""
+	@echo "$(CC_CYAN)Bundle$(CC_END)"
+	@echo "PACKAGE_NAME=$(PACKAGE_NAME)"
+	@echo "CHANNELS=$(CHANNELS)"
+	@echo "DEFAULT_CHANNEL=$(DEFAULT_CHANNEL)"
+	@echo "BUNDLE_IMAGE=$(BUNDLE_IMAGE)"
+ifneq ($(ADDITIONAL_BUNDLE_IMAGE),)
+ifneq ($(ADDITIONAL_BUNDLE_IMAGE_TAG),)
+	@echo "ADDITIONAL_BUNDLE_IMAGE=$(ADDITIONAL_BUNDLE_IMAGE)"
+endif
+endif
+	@echo ""
+	@echo "$(CC_CYAN)Catalog$(CC_END)"
+	@echo "CATALOG_IMAGE=$(CATALOG_IMAGE)"
+ifneq ($(ADDITIONAL_CATALOG_IMAGE),)
+ifneq ($(ADDITIONAL_CATALOG_IMAGE_TAG),)
+	@echo "ADDITIONAL_CATALOG_IMAGE=$(ADDITIONAL_CATALOG_IMAGE)"
+endif
+endif
+	@echo ""
+	@echo "$(CC_CYAN)Other$(CC_END)"
+	@echo "OS=$(OS)"
+	@echo "ARCH=$(ARCH)"
+	@echo "DATE=$(DATE)"
+	@echo "$(CC_YELLOW)==============================================$(CC_END)"
+
+
+########## Tools
+
+
+define target_dir
+    @{ \
+		if [ -d $(1) ]; \
+		then \
+			if  [[ $(1) == target/* ]]; \
+			then \
+				rm -rf "$(1)"; \
+			fi; \
+		fi; \
+		mkdir -p $(1); \
+   	}
+endef
+
+
+OPERATOR_SDK_VERSION ?= 1.37.0
+OPERATOR_SDK = $(shell pwd)/target/bin/operator-sdk
+.PHONY: install-operator-sdk
+install-operator-sdk: ## Install operator-sdk v1.37.0
+	@{ \
+		if [ ! -f $(OPERATOR_SDK) ]; \
+		then \
+			mkdir -p $(dir $(OPERATOR_SDK)); \
+			curl -sSLo $(OPERATOR_SDK) "https://github.com/operator-framework/operator-sdk/releases/download/v$(OPERATOR_SDK_VERSION)/operator-sdk_$(OS)_$(ARCH)"; \
+			chmod +x $(OPERATOR_SDK); \
+		fi; \
+	}
+
+
+.PHONY: install-opm
+OPM_VERSION ?= 1.48.0
+OPM = $(shell pwd)/target/bin/opm
+install-opm: ## Install opm v1.48.0
+	@{ \
+		if [ ! -f $(OPM) ]; \
+		then \
+			mkdir -p $(dir $(OPM)); \
+			curl -sSLo $(OPM) "https://github.com/operator-framework/operator-registry/releases/download/v$(OPM_VERSION)/$(OS)-$(ARCH)-opm"; \
+			chmod +x $(OPM); \
+		fi; \
+	}
+
+
+YQ_VERSION ?= 4.27.5
+YQ = $(shell pwd)/target/bin/yq
+.PHONY: install-yq
+install-yq: ## Install yq v4.27.5
+	@{ \
+		if [ ! -f $(YQ) ]; \
+		then \
+			mkdir -p $(dir $(YQ)); \
+			curl -sSLo $(YQ) "https://github.com/mikefarah/yq/releases/download/v$(YQ_VERSION)/yq_$(OS)_$(ARCH)"; \
+			chmod +x $(YQ); \
+		fi; \
+	}
+
+
+########## Targets
+
+
+.PHONY: build
+build: ## TODO
+	mvn clean install $(BUILD_OPTS)
+
+
+.PHONY: image-build
+image-build: ## TODO
+	docker build -f controller/src/main/docker/Dockerfile.jvm -t $(IMAGE) controller/target
+ifneq ($(ADDITIONAL_IMAGE),)
+ifneq ($(ADDITIONAL_IMAGE_TAG),)
+	docker tag $(IMAGE) $(ADDITIONAL_IMAGE)
+endif
+endif
+
+
+.PHONY: image-push
+image-push: ## TODO
+	docker push $(IMAGE)
+ifneq ($(ADDITIONAL_IMAGE),)
+ifneq ($(ADDITIONAL_IMAGE_TAG),)
+	docker push $(ADDITIONAL_IMAGE)
+endif
+endif
+
+
+.PHONY: image
+image: image-build image-push ## TODO
+
+
+.PHONY: deploy
+deploy: ## Deploy the Operator to a cluster using kubectl
+	kubectl create namespace $(NAMESPACE) || true
+	kubectl kustomize controller/src/main/deploy/install | envsubst | kubectl -n $(NAMESPACE) apply -f -
```

Comment:
They say `except the environment variable cannot have underscores otherwise the replacement does not happen.`. I'm open to playing with this, but after the release.